static/locales/en-US.yaml

```diff
@@ -111,6 +116,7 @@ Search Filters:
     Location: Location
     HDR: HDR
     VR180: VR180
+    New: New
```

Comment:
Was this accidentally added? I don't see any changes to the search filters code to use this string.

Reply:
Oh yeah, I will remove that